.eslintrc.js

```diff
@@ -1,5 +1,10 @@
 module.exports = {
-  extends: ['@nimblehq/eslint-config-nimble-react', '@nimblehq/eslint-config-nimble-testing'],
+  extends: [
+    '@nimblehq/eslint-config-nimble-react',
+    '@nimblehq/eslint-config-nimble-testing',
+    'plugin:cypress/recommended',
+    'prettier',
```

Comment:
we can remove it as already added in `@nimblehq/eslint-config-nimble-react`, we don't need to override